doc/specs/cli/apps.md

```diff
@@ -61,38 +61,31 @@ Transaction submitted with hash: <Transaction Hash>
 pocket apps create-aat <appAddr> <clientPubKey>
 ```
 
-Creates a signed application authentication token \(version `0.0.1` of the AAT spec\), that can be embedded into
-application software for Relay servicing. Will prompt the user for the `<appAddr>` account passphrase.
+This CLI is hard-coded to generate an AAT with spec version 0.0.1.
```

Comment:
Please update this to match the actual command output.

Reply:
This was the output. How should I update the docs?

```bash
 pocket apps create-aat 0551ce49b10bd603c85b8de89434686efdbb9c7a aaa
2024/03/11 19:47:48 Initializing Pocket Datadir
2024/03/11 19:47:48 datadir = /Users/olshansky/.pocket
Enter passphrase:
{
  "version": "0.0.1",
  "app_pub_key": "1789ebc8b3a37ebb07b6a9bd8716f34a67004c61f2120a8c1f76560474d8e4f0",
  "client_pub_key": "aaa",
  "signature": "8f07d21bc90fe571fcc326d0ba281eb71f61ddc781abe8f125ebe27ea74a3a55c09f7e4c19744dc7488b362a022e767b0d4e57965c0461dfc97f11c0996d2101"
}
```